README.md

```diff
@@ -38,79 +32,409 @@ IncludeOS is free software, with "no warranties or restrictions of any kind".
-2. Update the [CMakeLists.txt](./seed/service/CMakeLists.txt) to specify the name of your project, enable any needed drivers or plugins, etc.
+Now that you have all your build requirements ready, you can start the service.
 
-**Example:**
+However to start a service a good practice is to always activate the service
+requirements;
 
 ```
-    $ cp -r seed/service ~/my_service
-    $ cd ~/my_service
-    $ emacs service.cpp
-    ... add your code
-    $ mkdir build && cd build
-    $ cmake ..
-    $ make
-    $ boot my_service
+  $ source activate.sh
 ```
 
-Take a look at the [examples](./examples) and the [tests](./test). These all started out as copies of the same seed.
+Now you can boot the demo service with `boot <service-name>`.
+
+```
+  $ boot demo
+```
+
+This should start your demo service and show something along the following lines:
+
+```
+  ================================================================================
+  IncludeOS  (x86_64 / 64-bit)
+  +--> Running [ IncludeOS Demo Service ]
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  Service started
+  Made device_ptr, adding to machine
+    [ Network ] Creating stack for VirtioNet on eth0 (MTU=1500)
+       [ Inet ] Bringing up eth0 on CPU 0
+  *** Basic demo service started ***
+```
+
+To build more examples follow the [README](https://github.com/includeos/demo-examples/blob/master/README.md) in the demo-examples repository.
+
+##### Writing your first service
+
+To start writing your first service,
+- Copy the [demo_service/service.cpp](https://github.com/includeos/demo-examples/blob/master/demo_service/service.cpp) example.
+- Then start implementing the `Service::start` function in the `Service` class, located in `<your-service-name>/service.cpp` (very simple example provided). This function will be called once the OS is up and running.
+- Update the `CMakeLists.txt` to specify the name of your project, enable any needed drivers or plugins, etc.
+- Update the `conanfile.txt` to specify any build requirements and generators.
+
+You should then be able to run your service in the same way as the demo_service.
+
+To add a new service to our demo examples, make a PR to our [demo-examples](https://github.com/includeos/demo-examples) repo. Make sure to
+add a README in your example folder with description of your service.
+
+**Note:** Remember to deactivate your service environment after your work with:
+
+```
+  $ source deactivate.sh
+```
+
+<!-- TODO: ### How building with Conan works? -->
+___
+
+### Getting IncludeOS in editable mode
+
+To get started with getting the conan package in editable mode,
```

Comment:
Maybe we should explain here why you'd want to do this -- this is for kernel development?

Reply:
Yes, we need to add why this feature is useful to kernel developers and how it works.
ref: https://docs.conan.io/en/latest/developing_packages/editable_packages.html